code_generator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -30,6 +30,81 @@ namespace <#= qualifiedNamespace #>
         }
 <# } #>
 
+
```

Comment:
Little hard to read. Do we have an example?

Reply:
Ah yes - will post one shortly